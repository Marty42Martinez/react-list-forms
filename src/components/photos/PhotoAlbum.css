@import url('https://fonts.googleapis.com/css?family=Pacifico');

.PhotoAlbum {
  width: 95vw;
  padding: 1rem;
  margin: 0;

  h2 {
    padding: 1rem;

    font-family: 'Pacifico', cursive;
    font-size: 2rem;
    text-align: center;
  }
  ul {

    display: flex;
    flex-direction: row;
    justify-content: space-around;
    flex-wrap: wrap;
    list-style-type: none;
  }
  li {
    height: 10rem;
    margin: 5px;
  }

  img {
    width: auto;
    height: 100%;
  }

}