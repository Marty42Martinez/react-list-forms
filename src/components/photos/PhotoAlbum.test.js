import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('Photo Album component', () => {
  it('renders a PhotoAlbum', () => {
    const photos = [
      'coolImg1',
      'hotImg2',
      'lukewarmImg3'
    ];
    const wrapper = shallow(<PhotoAlbum title='Dope Album' photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
