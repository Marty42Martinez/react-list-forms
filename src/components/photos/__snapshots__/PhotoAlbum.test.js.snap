// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Album component renders a PhotoAlbum 1`] = `
<section
  className="PhotoAlbum"
>
  <h2>
    Dope Album
  </h2>
  <Photos
    photos={
      Array [
        "coolImg1",
        "hotImg2",
        "lukewarmImg3",
      ]
    }
  />
</section>
`;
